bundles/org.openhab.binding.radiothermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -119,13 +139,19 @@
 		<item-type>Number</item-type>
 		<label>Status</label>
 		<description>Indicates the Current Running Status of the HVAC System</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
 		<state min="0" max="2" pattern="%d"/>
 	</channel-type>
 
-	<channel-type id="fan_status">
+	<channel-type id="fanStatus">
 		<item-type>Number</item-type>
 		<label>Fan Status</label>
 		<description>Indicates the Current Fan Status of the HVAC System</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
```

Comment:
Is this a read-only or a writable channel and what is this for kind of status?
I ask this to determine the appropiate tags.
For reference: https://github.com/openhab/openhab-addons/issues/12262

Reply:
Read-only. It indicates if the furnace or air conditioner is currently operating

Reply:
AFAIK a `Point` should always be combined with a `Property` tag. I guess this can be mode? As it has three states?